src/infer_response.h

```diff
@@ -61,6 +61,10 @@ class InferenceResponseFactory {
         alloc_userp_(alloc_userp), response_fn_(response_fn),
         response_userp_(response_userp), response_delegator_(delegator),
         is_cancelled_(false)
+#ifdef TRITON_ENABLE_STATS
+        ,
+        response_stats_index_(0)
```

Comment:
did pre-commit hook put comma and parameter on two lines?

Reply:
Yes, pre-commit did it.